source/docs/3/digitizing_basics.rst

```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
 
+ .. image:: /static/3/digitizing_basics/images/14.png
+    :align: center   
+ 
+15. In the :guilabel:`Layer Styling Panel`, search for different `road` layer styles. Select ``topo road``. Click :guilabel:`OK`.
+
+ .. image:: /static/3/digitizing_basics/images/15.png
+    :align: center
+   
+16. Now the road layer will be clearly visible. If you are satisfied with the work, click :guilabel:`Save Layer Edits` button to save the changes. 
+
+ .. image:: /static/3/digitizing_basics/images/16.png
+    :align: center
+	
+17. Before we digitize the remaining roads, it is essential to update some other important snap settings to create an error-free layer. Right-click on any space on the toolbar area and activate the :guilabel:`Snapping toolbar.` 
+
+ .. image:: /static/3/digitizing_basics/images/17.png
+    :align: center
+   
+18. Now an :guilabel:`Enable Snapping` (Magnet Icon) will appear on the panel. Click on it to enable it and select :guilabel:`All Layers` and choose ``Open Snapping Options..``. 
+
+ .. image:: /static/3/digitizing_basics/images/18.png
+    :align: center 
+	
+19. In the :guilabel:`Snapping options` dialog, click the :guilabel:`Snapping on Intersection`, which allows you to snap on an intersection of a background layer.
+
+ .. image:: /static/3/digitizing_basics/images/19.png
+    :align: center
+   
+20. Now you can click :guilabel:`Add feature` button and digitize other roads around the park. Make sure to click :guilabel:`Save Edits` after adding a new feature to save your work. A helpful tool to help you digitize is the **Vertex Tool**. Click the :guilabel:`Vertex Tool` button and select ``Vertex Tool (Current Layer)``.
+
+ .. image:: /static/3/digitizing_basics/images/20.png
+    :align: center       
+	
+21. Once the node tool is activated, click on any feature to show the vertices. Click on any vertex to select it. The vertex will change the color once it is selected. Now you can click and drag your mouse to move the vertex. This is useful when you want to make adjustments after the feature is created. You can also delete a selected vertex by clicking the :kbd:`Delete` key.    (:kbd:`Option+Delete` on a mac)
+
+ .. image:: /static/3/digitizing_basics/images/21.gif
+    :align: center
+   
+22. Once you have finished digitizing all the roads, click the :guilabel:`Toggle Editing` button. Click :guilabel:`Save`.
+
+ .. image:: /static/3/digitizing_basics/images/22.png
+    :align: center
+   
+23. In the :guilabel:`New GeoPackage Layer` dialog, click the :guilabel:`...` button and select the GeoPackage database named ``digitizing.gpkg``. Name the new layer as an attribute called ``Parks``. Select ``MultiPolygon`` as the :guilabel:`Type`. The base topographic map is the ``EPSG:2193 - NZGD 2000`` CRS. Click :guilabel:`OK`. In :guilabel:`New Field` Enter ``Name``, and the :guilabel:`type` as :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length` and click :guilabel:` Add to Fields List.`. Click :guilabel:`OK`. 

```

Comment:
Now we will create another layer to digitize the parks as polygons.

Reply:
Done.